### PR TITLE
Task #209920 Task #209959 Task #209928  [WEB] Corrections in ui in camp execution and navigations 

### PR DESCRIPTION
**Pull request by chaitanyakole**
### I have ensured that following `Pull Request Checklist` is taken care of before sending this PR
* [x] Code is formatted as per format decided
* [x] Updated acceptance criteria in tracker
* [NA] Updated test cases in test-cases-tracker
https://tracker.tekdi.net/issues/209920
https://tracker.tekdi.net/issues/209928
https://tracker.tekdi.net/issues/209959

### CODE REVIEW

**Review by swapnilphalke03 — APPROVED** [review]

File: lib/common-lib/src/components/IconByName.js [hunk 1]
```diff
@@ -145,14 +146,19 @@ function IconButton({ icon, isDisabled, prefix, ...props }) {
   }
 }
 
-function IconByName({ _icon, ...props }) {
+function IconByName({ _icon, _size, ...props }) {
   let icon
 
   if (!_icon?.size) {
-    _icon = { ..._icon, size: 24 }
+    _icon = { ..._icon, size: _size || 24 }
```
Comment (sagarkoshti1990) [inline lib/common-lib/src/components/IconByName.js:153]: remove this _size prop and use _icon to set size

**Review by sagarkoshti1990 — CHANGES_REQUESTED** [review]

File: apps/front-end/src/pages/front-end/Camp/CampExecution/CampExecution.js [hunk 2]
```diff
@@ -268,18 +268,36 @@ export default function CampExecution({ footerLinks }) {
...
-            <FrontEndTypo.Primarybutton onPress={campBegin}>
-              {t("YES_ABSOLUTELY")}
-            </FrontEndTypo.Primarybutton>
-            <FrontEndTypo.Secondarybutton
-              onPress={(e) => navigate(`/camps/${id}/campotherplans`)}
-            >
-              {t("NO_PLAN")}
-            </FrontEndTypo.Secondarybutton>
+            {todaysData?.[0]?.end_date === null ? (
+              <>
```
Comment (sagarkoshti1990) [inline apps/front-end/src/pages/front-end/Camp/CampExecution/CampExecution.js:273]: please don't use fragment.

File: apps/front-end/src/pages/front-end/Camp/CampTodayActivities.js [hunk 3]
```diff
@@ -81,54 +87,77 @@ export default function CampTodayActivities({ footerLinks, setAlert }) {
...
+                alignSelf={"center"}
+                w="75px"
+                h="60px"
+              />
+
               <FrontEndTypo.H2 color="textMaroonColor.400">
                 {t("LEARNING_ACTIVITIES")}
               </FrontEndTypo.H2>
+              {sessionList === true && (
+                <CheckCircleIcon size="36" color="successColor" />
```
Comment (sagarkoshti1990) [inline apps/front-end/src/pages/front-end/Camp/CampTodayActivities.js:113]: use `iconByName`

File: apps/front-end/src/pages/front-end/Camp/CampTodayActivities.js [hunk 4]
```diff
@@ -81,54 +87,77 @@ export default function CampTodayActivities({ footerLinks, setAlert }) {
...
+                resizeMode="contain"
+                alignSelf={"center"}
+                w="75px"
+                h="60px"
+              />
               <FrontEndTypo.H2 color="textMaroonColor.400">
                 {t("MISCELLANEOUS_ACTIVITIES")}
               </FrontEndTypo.H2>
+              {selectValue?.[0] && (
+                <CheckCircleIcon size="36" color="successColor" />
```
Comment (sagarkoshti1990) [inline apps/front-end/src/pages/front-end/Camp/CampTodayActivities.js:144]: use `iconByName`

**Review by sagarkoshti1990 — CHANGES_REQUESTED** [review]

**Pull request merged**